DQM package compiler warning

I noticed the following warning message when building the HEAD of the _develop_ branch in the _dqm_ repo today:

```
[174/179] Building CXX object dqm/CMakeFiles/dqm_DQMProcessor_duneDAQModule.dir/plugins/DQMProcessor.cpp.o
/home/biery/dunedaq/26JulWorkDir_N0725Base/sourcecode/dqm/plugins/DQMProcessor.cpp: In member function ‘void dunedaq::dqm::DQMProcessor::RequestMaker()’:
/home/biery/dunedaq/26JulWorkDir_N0725Base/sourcecode/dqm/plugins/DQMProcessor.cpp:152:7: warning: missing initializer for member ‘dunedaq::dqm::DQMProcessor::RequestMaker()::AnalysisInstance::running_thread’ [-Wmissing-field-initializers]
       };
       ^
/home/biery/dunedaq/26JulWorkDir_N0725Base/sourcecode/dqm/plugins/DQMProcessor.cpp:152:7: warning: missing initializer for member ‘dunedaq::dqm::DQMProcessor::RequestMaker()::AnalysisInstance::name’ [-Wmissing-field-initializers]
```

It would be great if this could be fixed.

--- a/plugins/DQMProcessor.cpp
+++ b/plugins/DQMProcessor.cpp
@@ -148,8 +148,7 @@ DQMProcessor::RequestMaker()
       TLOG() << "ALGORITHM already running";
       map[std::chrono::system_clock::now() +
           std::chrono::milliseconds(static_cast<int>(fr->second.default_unavailable_time) * 1000)] = {
-        algo, fr->second.between_time, fr->second.default_unavailable_time
-      };
+        algo, fr->second.between_time, fr->second.default_unavailable_time, previous_thread, fr->second.name};
       map.erase(fr);
       continue;
     }

